src/main/java/models/Lottery.java

```diff
@@ -0,0 +1,13 @@
+package models;
+
+import core.basesyntax.ColorSupplier;
+import java.util.Random;
+
+public class Lottery {
+    public static Ball getRandomBall() {
+        ColorSupplier colorSupplier = new ColorSupplier();
+        String color = colorSupplier.getRandomColor();
+        int number = new Random().nextInt(100);
```

Comment:
make ColorSupplier and Random as class fields

Reply:
changed. but i wasn't sure if i had to make constructor here - and didn't do it